src/playermat/Playmat.ttslua

```diff
@@ -536,7 +536,7 @@ end
 
 function maybeUpdateActiveInvestigator(card)
   local notes = JSON.decode(card.getGMNotes())
-  if notes ~= nil and notes.type == "Investigator" and notes.id ~= activeInvestigatorId then
+  if notes ~= nil and notes.type == "Investigator" and notes.id ~= nil and notes.id ~= activeInvestigatorId then
```

Comment:
If you drop an investigator with data then an investigator that has data but no ID, this will leave the old stats, right?

Reply:
Yep, exactly

Reply:
Added handling to load default values for investigators without metadata